src/main/resources/db/migration/V1__Init_tables.sql

```diff
@@ -0,0 +1,141 @@
+DROP TABLE IF EXISTS cc_employee CASCADE;
+DROP TYPE IF EXISTS cc_employee_status;
+DROP TYPE  IF EXISTS cc_employee_role;
+
+CREATE TYPE cc_employee_status as enum ('INACTIVE','ACTIVE','ARCHIVED');
+CREATE TYPE cc_employee_role as enum ('STAFF','ADMIN');
+
+CREATE TABLE cc_employee
+(
+    id         BIGINT                   NOT NULL PRIMARY KEY,
+    email      VARCHAR(100)             NOT NULL,
+    password   CHAR(64)                 NOT NULL,
+    first_name VARCHAR(100)             NOT NULL,
+    last_name  VARCHAR(100)             NOT NULL,
+    role       cc_employee_role         NOT NULL,
+    created_at TIMESTAMP WITH TIME ZONE NOT NULL,
+    updated_at TIMESTAMP WITH TIME ZONE NOT NULL,
+    status     cc_employee_status       NOT NULL
+);
+
+COMMENT ON COLUMN cc_employee.email IS 'Employee''s email as login username';
```

Comment:
Why do we need ' ' here?

Reply:
Because we want to insert a single quote to the comment, we need to use '' instead of '.